Makefile

```diff
@@ -2,6 +2,7 @@ MODULE   = $(shell env GO111MODULE=on $(GO) list -m)
 DATE         ?= $(shell date +%FT%T%z)
 KO_DATA_PATH  = $(shell pwd)/cmd/$(TARGET)/kodata
 TARGET        = kubernetes
+CR            = config/default
```

Comment:
I don't see `default` inside `config` :thinking:

Reply:
it is at `config/crs/{kubernetes,openshift}/config/default`